docs/build-guides/mac-build.md

```diff
@@ -8,7 +8,7 @@ Automated build: clone the repo and run `mac.build.sh` in the `libindy` folder.
 2. Install required native libraries and utilities (libsodium is added with URL to homebrew since version<1.0.15 is required)
    ```
    brew install pkg-config
-   brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/65effd2b617bade68a8a2c5b39e1c3089cc0e945/Formula/libsodium.rb   
+   brew install libsodium   
```

Comment:
I do not know the general status.  I asked on the indy-sdk rocket chat channel and Tomislav replied that the old version was no longer needed for MAC and since the old version was causing troubles (failing to install the indy-cli while trying to use the old command) I thought I would try and get this page updated to something that works.